test/data/html.js

```diff
@@ -31,6 +32,48 @@ exports.test_data = {
             { fragment: '\n', output: '{{eof}}' }
         ],
     }, {
+        name: "Custom Extra Liners (empty)",
+        description: "",
+        matrix: [
+            {
+                options: [
+                    { name: "extra_liners", value: "[]" }
+                ]
+            },
+
+        ],
+        tests: [
+            { fragment: '<html><head><meta></head></html>', output: '<html>\n<head>\n    <meta>\n</head>\n</html>' }
+        ],
+    }, {
+        name: "Custom Extra Liners (default)",
+        description: "",
+        matrix: [
+            {
+                options: [
+                    { name: "extra_liners", value: "null" }
+                ]
+            },
+
+        ],
+        tests: [
+            { fragment: '<html><head></head><body></body></html>', output: '<html>\n\n<head></head>\n\n<body></body>\n\n</html>' }
+        ],
+    }, {
+        name: "Custom Extra Liners (p)",
+        description: "",
+        matrix: [
+            {
+                options: [
+                    { name: "extra_liners", value: "['p', '/p']" }
+                ]
+            },
+
+        ],
+        tests: [
+            { fragment: '<div><p>x</p></div>', output: '<div>\n\n    <p>x\n\n    </p>\n</div>' }
```

Comment:
Have this input be the same as the one for the other tests.  That will show that setting this makes the existing extra liners not extra anymore.